pkgs/tools/networking/curl/default.nix

```diff
@@ -67,10 +72,11 @@ stdenv.mkDerivation rec {
       # to nss-cacert from the default profile.
       "--without-ca-bundle"
       "--without-ca-path"
-      "--with-ca-fallback"
       "--disable-manual"
+      ( if sslSupport || gnutlsSupport then "--with-ca-fallback" else "--without-ca-fallback")
```

Comment:
Doesn't this also apply for the wolfssl backend?

Reply:
The issue here is that wolfssl does not build with `--with-ca-fallback`, so we only pass `--with-ca-fallback` for openssl and gnutls.